src/components/TopBar/TopBar.js

```diff
@@ -23,24 +25,17 @@ const useStyles = makeStyles(theme => ({
   },
 }));
 
-const setPageActionHooks = data => {
-  return {
-    type: CHANGE_PAGE_ACTION,
-    data,
-  };
-};
+const TopBarWrapper = styled.div`
+  display: flex;
+  height: 5rem;
+`;
 
-export function TopBarComponent() {
+export function TopBar() {
```

Comment:
Why renaming component name?
We could rename all components from `...Component` to `...`, if we agreed in that task were we discussed it.
But renaming only a few components, while leaving the other components with the old names is messy: you're breaking the convention, the code style. Now we're going to have `SearchPageComponent.js` but `TopBar.js`.

The [README](https://github.com/DarthVanger/NeuralNotes#structure) still says

>All files are using their type as filename postfix.

You're just making it more confusing for the other devs by renaming `TopBarComponent.js` to `TopBar.js`...

We should either change all the component names or change nothing... Otherwise it's a mess...

Reply:
I renaming components that I refactored so it's easier to distinguish them.

Please concentrate on making successfull app rather than fighting over convention or stylecode.

It's easier to read and write short names, and it's really remoralizing to write long names as <ComponentNameComponent>. I just try to do productive work, so you and your app are successful and I'm comfortable to do that work